devito/types/dimension.py

```diff
@@ -742,6 +746,10 @@ def free_symbols(self):
             retval |= self.condition.free_symbols
         return retval
 
+    @cached_property
+    def _indexify_map(self):
+        return {self.spacing: 1, -self.spacing: -1}
```

Comment:
why is this different than the superclass'?

Reply:
Super-class is `{self.spacing: 1}`